src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  background-color: black;
+}
+
+.txt {
+  background-color: #ddd;
+  border: 1px solid #e6635f;
+  width: 284px;
+  margin: 0;
+  padding: 10px;
+}
+
+.top,
+#qa-block {
+  background-color: #308dc0;
+  border: 1px solid #e6635f;
+  width: 328px;
+  padding: 10px;
+  margin: 20px 12px;
+}
+
+.top {
+  margin: 40px 12px;
+}
+
+.mid {
+  background-color: orange;
+  border: 1px solid #e6635f;
```

Comment:
.txt, .mid and  .top have same border and padding properties, move them to separate class

Reply:
move border property to `.pad` class, why you moved only padding?